backend/src/main/java/quarano/department/web/TrackedCaseController.java

```diff
@@ -225,6 +231,24 @@
 				.concludeIfValid(ResponseEntity::ok);
 	}
 
+	@DeleteMapping("/hd/cases/{identifier}/account")
```

Comment:
I ask myself if this is right here or rather belongs in the `RegistrationController`. There probably wouldn't be so many imports added.